server/fleet/hosts.go

```diff
@@ -220,6 +220,9 @@ type MDMHostData struct {
 	// ServerURL is the server_url stored in the host_mdm table, loaded by
 	// JOIN in datastore
 	ServerURL *string `json:"server_url" db:"-" csv:"mdm.server_url"`
+	// Name is the name of the MDM solution for the host.
+	Name string `json:"name" db:"name" csv:"-"`
```

Comment:
This will not be a column available in the frontend? (if it is, I think it'd need to be added to the CSV output).

Reply:
my understanding is that this is just to allow the UI to do `mdm.name == "Fleet"`